frontend/src/components/CanteenIndicators.vue

```diff
@@ -1,25 +1,33 @@
 <template>
-  <div>
-    <p :class="{ 'my-0': true, inline: singleLine }" v-if="hasSatelliteCanteens">
-      <v-icon small>$community-fill</v-icon>
-      {{ canteen.satelliteCanteensCount }} satellites
+  <div
+    :class="{
+      'grey--text text--darken-2': true,
+      'd-flex flex-wrap': singleLine,
+      'fr-text': !dense,
+      'fr-text-xs': dense,
+    }"
+  >
+    <p :class="{ 'my-0 d-flex align-center': true, 'inline mr-3': singleLine }" v-if="canteen.city">
+      <v-icon :small="!dense" :x-small="dense" class="mr-1">$map-pin-2-line</v-icon>
+      {{ canteen.city }}
     </p>
-    <p :class="{ 'my-0': true, inline: singleLine }" v-if="hasDailyMealCount">
-      <span class="mx-1" v-if="singleLine && hasSatelliteCanteens">/</span>
-      <v-icon small aria-hidden="false" role="img" aria-label="Repas">$restaurant-fill</v-icon>
+    <p :class="{ 'my-0 d-flex align-center': true, 'inline mr-3': singleLine }" v-if="hasDailyMealCount">
+      <v-icon :small="!dense" :x-small="dense" class="mr-1" aria-hidden="false" role="img">$team-line</v-icon>
       <!-- eslint-disable-next-line prettier/prettier-->
-      {{ canteen.dailyMealCount }} par jour<span v-if="canteen.productionType === 'site_cooked_elsewhere'">, livrés</span>
+      {{ canteen.dailyMealCount }} couverts<span v-if="canteen.productionType === 'site_cooked_elsewhere'">, livrés</span>
```

Comment:
C'est p-e mieux sans la virgule... çad « _x couverts livrés_ » et non pas « _x couverts, livrés_ »
```suggestion
      {{ canteen.dailyMealCount }} couverts<span v-if="canteen.productionType === 'site_cooked_elsewhere'"> livrés</span>
```

Reply:
Je l'enleve maintenant qu'on a ajouté le tag "Cantine satellite" sur les cartes